qrand.go

```diff
@@ -68,25 +67,20 @@ func Read(buf []byte) (n int, err error) {
 	return Reader.Read(buf)
 }
 
-type ΨSource struct{}
+type Source struct{}
 
-var qSource ΨSource
+//Seed takes no argument because there is nothing to seed
+func (s *Source) Seed(seed int64) {}
 
-func (s *ΨSource) Seed(seed int64) {}
-
-func (s *ΨSource) Uint64() (value uint64) {
+func (s *Source) Uint64() (value uint64) {
```

Comment:
We should comment all public methods, shouldn't we? Use the comments on the equivalent methods in the `rand` library as inspiration!